### PR TITLE
Fix some minor issues after review

### PR DESCRIPTION
**Pull request by Alroniks**

### CODE REVIEW

**Comment by Alroniks** [comment]
Ups, I've broken tests. Fixing now

**Comment by Alroniks** [comment]
@krampstudio Sorry for my rush, fixed tests.

**Comment by jsconan** [comment]
Cool, the mass actions work again (multi selection).

File: actions/class.Users.php [hunk 1]
```diff
@@ -256,11 +256,14 @@ public function checkLogin(){
...
-        if(!$this->hasRequestParameter('uri')){
+        if (!$this->hasRequestParameter('uri')) {
             throw new Exception('Please set the user uri in request parameter');
         }
 
-        $user = new core_kernel_classes_Resource(tao_helpers_Uri::decode($this->getRequestParameter('uri')));
+        $uri = $this->getRequestParameter('uri');
+        $uri = is_array($uri) ? array_shift($uri) : $uri;
+
+        $user = new core_kernel_classes_Resource(tao_helpers_Uri::decode($uri));
```
Comment (jsconan) [inline actions/class.Users.php]: Why change the webservice? Isn't be better to change the javascript action to extract the array before requesting the server?
I think it is preferable to keep webservices consistent, and adapt the clients to work with.
@krampstudio what do you think about that?

Comment (Alroniks) [inline actions/class.Users.php]: It's a feature of mass actions. When using selection and mass actions, component pass to controller array of fields. In this case array of URIs (uri[] = something). And actions should have ability process this cases, IMO.

Comment (krampstudio) [inline actions/class.Users.php]: For me we must differentiate the 2 cases:
 - using an action or a "selection by row" we should get only the selected item
 - using the selection by checkboxes we always have an array of item

Then in the user table, the edit action is always on one user, so the controller shouldn't take care of the arrays (this controller expects an uri as a string, if you send him something else, then it's not valid).

**Comment by krampstudio** [comment]
By the way, could you change  the class `selectablebyclick` by `hoverable` or `selectable` 
and also the property name `selectbyclick` by `rowSelection` or something similar

**Comment by Alroniks** [comment]
Fixed names of classes in styles, and also was reverted changes in users controller. Summarizing previous discussions, now it works as expected and the developer responsible for proper use actions (especially mass actions).

**Pull request merged**